ployst/github/test/test_end_to_end.py

```diff
@@ -15,7 +15,7 @@ class TestEndToEnd(TestCase):
     def setUp(self):
         ensure_dummy_clone_available()
 
-    @patch(__name__ + '.tasks.hierarchy.client', MockClient())
+    @patch('ployst.github.tasks.hierarchy.client', MockClient())
```

Comment:
I had patched this like this so that github worked as a standalone app (rather than being reliant on being inside ployst).

I don't mind us losing that though.